### ISSUE TITLE
librocm_smi detection/install detection issues

### ISSUE THREAD

**Issue by pszi1ard** [ROCm-Developer-Tools/ROCmValidationSuite#239]
If I use a directory layout different from the guide's suggestion, the rocm_smi_lib library path is not correctly detected -- the cmake scripts seem to assume that the library is in ``../build``.

Furthermore, when using the guide's commands and directory layout as is (or fixing the detection by manually passing the lib path to cmake), the resulting deb does not contain librocm_smi64.so:

```
$ dpkg -c ./rocm-validation-suite-0.0.16.deb | grep rocm_smi -c 
0
```

Consequently, after install:
```
$ /opt/rocm/rvs/rvs 
[RESULT] [558156.110074] action_1 gpup 59231 mem_banks_count 1
[RESULT] [558156.110125] action_1 gpup 59231 0 version_major 0
[RESULT] [558156.110197] action_1 gpup 57178 mem_banks_count 1
[RESULT] [558156.110225] action_1 gpup 57178 0 version_major 0
[RESULT] [558156.110341] action_2 gpup 59231 mem_banks_count 1
[RESULT] [558156.110422] action_2 gpup 57178 mem_banks_count 1
[RESULT] [558156.152361] [action_3] peqt FALSE
RVS-CLI: could not load .so '/opt/rocm/rvs//libgm.so.0' reason: librocm_smi64.so: cannot open shared object file: No such file or directory
RVS-CLI: module 'gm' not available.
RVS-CLI: action 'action_4' could not crate action object in module 'gm'
```

**Comment by mlucinhdl** [comment]
Fix/clarification merged in #240

**Comment by pszi1ard** [comment]
Thanks for the fixes, good to see quick action.
